core-bundle/src/Resources/contao/templates/backend/be_tinyMCE.html5

```diff
@@ -37,15 +38,15 @@ window.tinymce && tinymce.init({
         }
       });
     },
-    file_picker_types: <?= json_encode($this->fileBrowserTypes) ?>,
+    file_picker_types: '<?= implode(' ', $this->fileBrowserTypes) ?>',
```

Comment:
Was this wrong ever since? 

Judging by the [docs](https://www.tiny.cloud/docs-4x/configure/file-image-upload/#file_picker_types ), in v4 this should also have been "types of file pickers you need by a space or comma separated list of type names".

Reply:
Ah, good find. Then we can make `$this->fileBrowserTypes` a space separated string and it will be compatible with both TinyMCE 5 and 6. 🎉

Reply:
Adjusted in afb35bc10f026315e3cc3bcae5be8c2013b00d4a.